code/modules/mob/living/carbon/alien/organs.dm

```diff
@@ -113,12 +114,20 @@
 			owner.adjustPlasma(plasma_rate*multiplier)
 		else
 			var/heal_amt = heal_rate*multiplier
+
 			if(!isalien(owner))
 				heal_amt *= 0.2
-			owner.adjustBruteLoss(-heal_amt)
-			owner.adjustFireLoss(-heal_amt)
-			owner.adjustOxyLoss(-heal_amt)
-			owner.adjustCloneLoss(-heal_amt)
+
+			if (owner.lying)
+				var/heal_divider = owner.stat == UNCONSCIOUS ? 2 : 1
```

Comment:
if you mean aliens healing slowly when unconscious, it's to prevent them from healing while being gunned down. I recovered from crit in about 20 seconds while playing with Shamzie yesterday

If you mean non-passive healing, it's because it encourages aliens to camp in the hive and never attack off of weeds, where they have a massive disadvantage due to not being able to instantly recover from shots. Marines don't automatically heal, aliens shouldn't either. Alternatively, they could heal from minor injuries passively, as humies do?